src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -103,6 +113,30 @@ class ContactMethodDetailsPage extends Component {
         return decodeURIComponent(lodashGet(this.props.route, 'params.contactMethod'));
     }
 
+    /**
+     * Checks if the user is allowed to change their default contact method by looking at
+     * their security group settings on their primary domain.
+     *
+     * @returns {Boolean}
+     */
+    getCanChangeDefaultContactMethod() {
+        const domainName = Str.extractEmailDomain(this.props.session.email);
+        const primaryDomainSecurityGroupID = lodashGet(this.props.myDomainSecurityGroups, domainName);
+
+        // If there's no security group associated with the user for the primary domain,
+        // default to allowing the user to change their default contact method.
+        if (!primaryDomainSecurityGroupID) {
+            return true;
+        }
+
+        const hasRestrictedPrimaryLogin = lodashGet(this.props.securityGroups, [
```

Comment:
NAB: You could just `return lodashGet(…)` directly without assigning the variable.

Reply:
Done 👍